docs/docs/03-tools/07-gpt-file-reference.md

```diff
@@ -43,21 +43,25 @@ Tool instructions go here.
+| `Name`               | The name of the tool.                                                                                                                         |
+| `Model Name`         | The LLM model to use, by default it uses "gpt-4-turbo".                                                                                       |
+| `Global Model Name`  | The LLM model to use for all the tools.                                                                                                       |
+| `Description`        | The description of the tool. It is important that this properly describes the tool's purpose as the description is used by the LLM.           |
+| `Internal Prompt`    | Setting this to `false` will disable the built-in system prompt for this tool.                                                                |
+| `Tools`              | A comma-separated list of tools that are available to be called by this tool.                                                                 |
+| `Global Tools`       | A comma-separated list of tools that are available to be called by all tools.                                                                 |
+| `Parameter` / `Args` | Arguments for the tool. Each argument is defined in the format `arg-name: description`.                                                       |
+| `Max Tokens`         | Set to a number if you wish to limit the maximum number of tokens that can be generated by the LLM.                                           |
+| `JSON Response`      | Setting to `true` will cause the LLM to respond in a JSON format. If you set true you must also include instructions in the tool.             |
+| `Temperature`        | A floating-point number representing the temperature parameter. By default, the temperature is 0. Set to a higher number for more creativity. |
+| `Chat`               | Setting it to `true` will enable an interactive chat session for the tool.                                                                    |
+| `Credential`         | Credential tool to call to set credentials as environment variables before doing anything else. One per line.                                 |
+| `Agents`             | A comma-separated list of agents that are available to the tool.                                                                              | 
+| `Share Tools`        | A comma-separated list of tools that are shared by the tool.                                                                                  |
+| `Context`            | A comma-separated list of context tools available to the tool.                                                                                |
+| `Share Context`      | A comma-separated list of context tools shared by this tool with any tool including this tool in its context.                                 | 
 
```

Comment:
There's more here, but I'm not sure if we should put everything (e.g. temperature and max_tokens) here